client/src/components/Kanban.js

```diff
@@ -1,201 +1,182 @@
-              key={column.name}
-              tasks={this.props.tasks}
-            />
-          )
-        })}
-      </div>
-    )
-  }
+Kanban.propTypes = {
+  tasks: PropTypes.array.isRequired,
+  columns: PropTypes.array.isRequired
 }
 
-class Column extends React.Component {
-  static propTypes = {
-    name: PropTypes.string.isRequired,
-    tasks: PropTypes.array.isRequired,
-    taskUUIDs: PropTypes.array.isRequired
-  }
+const Column = props => {
+  const [open, setOpen] = useState(false)
+  const tasks = props.tasks.filter(
+    task => props.taskUUIDs.indexOf(task.uuid) > -1
+  )
+  const counters = tasks.reduce((counter, task) => {
+    counter[task.customFieldEnum1] = ++counter[task.customFieldEnum1] || 1
+    return counter
+  }, {})
 
-  constructor(props, context) {
-    super(props, context)
+  return (
+    <Panel style={{ flex: "1 1 0%", margin: "4px" }}>
+      <Panel.Heading>
+        <strong>
+          <em>{props.name} </em>
+        </strong>
+      </Panel.Heading>
+      <Panel.Body>
+        <Pie
+          width={70}
+          height={70}
+          data={counters}
+          label={`${tasks.length}`}
+          segmentFill={entity => {
+            const matching = Object.entries(
+              Settings.fields.task.customFieldEnum1.enum
+            ).filter(candidate => {
+              return candidate[0] === entity.data.key
+            })
+            return matching.length > 0 ? matching[0][1].color : "#bbbbbb"
+          }}
+          segmentLabel={d => d.data.value}
+        />
+        <br />
+        <EngagementTrends
+          newValue={tasks.reduce(
+            (accumulator, task) => accumulator + task.lastMonthReports.length,
+            0
+          )}
+          oldValue={tasks.reduce(
+            (accumulator, task) =>
+              accumulator + task.preLastMonthReports.length,
+            0
+          )}
+          totalValue={tasks.reduce(
+            (accumulator, task) => accumulator + task.allReports.length,
+            0
+          )}
+        />
+        <br />
+        <br />
+        <strong>{Settings.fields.task.longLabel}</strong>
+        {"  "}
+        <Button bsSize="xs" onClick={() => setOpen(!open)}>
+          <Glyphicon glyph={open ? "triangle-top" : "triangle-bottom"} />
+        </Button>
+        <br />
+        {open && tasks.map(task => <Card task={task} key={task.uuid} />)}
+      </Panel.Body>
+    </Panel>
+  )
+}
 
-    this.state = { open: false }
-  }
+Column.propTypes = {
+  name: PropTypes.string.isRequired,
+  tasks: PropTypes.array.isRequired,
+  taskUUIDs: PropTypes.array.isRequired
+}
 
-  render() {
-    const { open } = this.state
-    const tasks = this.props.tasks.filter(
-      task => this.props.taskUUIDs.indexOf(task.uuid) > -1
-    )
-    const counters = tasks.reduce((counter, task) => {
-      counter[task.customFieldEnum1] = ++counter[task.customFieldEnum1] || 1
-      return counter
-    }, {})
+const Card = props => {
```

Comment:
Prefer `const Card = ({ task }) => {` instead of referring to `props.something` everywhere.